pom.xml

```diff
@@ -43,8 +43,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.0.2</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
```

Comment:
Won't this break compatibility with <8? If we're not using any features from 8+, I don't see the need

Reply:
Even 8 is EOL, there's no point in supporting Java 6 and 7 imo. People who need Java 6 support can stay on le_java for the time being. The tests for them currently don't work either and I don't see a point in fixing tests for dead Java versions.

Reply:
Also Java 12 will actually not even run Java 6 bytecode anymore, as it turns out.